helpers/genNetCDF/Topography.py

```diff
@@ -1,3 +1,4 @@
+from urllib.parse import non_hierarchical
```

Comment:
This line can be removed since `non_hierarchical` isn't used anywhere.

Reply:
removed during conflict resolution when merging develop back in